specification/synapse/data-plane/Microsoft.Synapse/preview/2021-06-01-preview/entityTypes/Dataset.json

```diff
@@ -402,8 +402,8 @@
       "type": "object",
       "properties": {
         "type": {
-          "type": "object",
-          "description": "Type of dataset compression. Type: string (or Expression with resultType string)."
+          "type": "string",
```

Comment:
I believe you will also need to update the type of anything that inherits from `DatasetCompression` using the `allOf` keyword.  I was getting an AutoRest failure before I did this.

Reply:
I found a bug on my side - please disregard this comment.